suitcase/jsonl/__init__.py

```diff
@@ -186,6 +188,9 @@ def __call__(self, name, doc):
             self._output_file.flush()
         return name, doc
 
+    def stop(self):
```

Comment:
This must accept `doc` as an argument, even though it won't do anything with it.

Reply:
good catch, I have fixed it